appveyor.yml

```diff
@@ -0,0 +1,56 @@
+
+version: 0.5.{build}
```

Comment:
Is this appveyor version?

Reply:
it is, but it's arbitrary - it's just for discerning "build numbers" o appveyor